entrypoint.sh

```diff
@@ -10,8 +10,18 @@ OTS_SSL=${OTS_SSL:='true'}
 OTS_COLOR=${OTS_COLOR:='#dd4a22'}
 
 echo "##### Performing vars check #####"
-if [[ -z $OTS_DOMAIN || -z $OTS_MASTER_KEY || -z $OTS_REDIS_USER || -z $OTS_REDIS_HOST || -z $OTS_REDIS_PORT  ]]; then
+if [[ -z $OTS_DOMAIN || -z $OTS_MASTER_KEY || -z $OTS_REDIS_PW || -z $OTS_REDIS_HOST || -z $OTS_REDIS_PORT  ]]; then
   echo "Required variable is not defined!"
+  echo "current configuration: "
```

Comment:
I think this debug output should only be printed when debugging is enabled. Maybe move it to the $DEBUG == "true" section.
Also mention the debug parameter in the readme.